write-fonts/src/tables/cmap.rs

```diff
@@ -164,6 +164,35 @@ impl CmapSubtable {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum MappingError {
+    /// When a char is mapped to multiple distinct GlyphIds.
+    /// gid1 is less than gid2.
+    /// If there are multiple conflicting mappings, one is chosen arbitrarily.
```

Comment:
Rustdoc renders doc comments sort of like git renders commits; the first line is expected to be a summary, followed by a newline, and then additional docs follow that.

More concretely, the first line is rendered as a short description when listing items in a module or crate, and the longer docs are only rendered in the specific page for that item.